katello/hooks/pre_commit/14-cdn_setting.rb

```diff
@@ -0,0 +1,10 @@
+if module_enabled?('katello')
+  param = @kafo.config.answers['katello']['cdn_ssl_version']
```

Comment:
Untested, but I wonder whether just `param_value('katello', 'cdn_ssl_version')` works.

Reply:
It does not, I was talking to @ehelms about this yesterday, when I added the puppet pr and installed puppet-strings it went away from the param so @ehelms told me to use this variable instead.

Reply:
Would you mind adding that as a comment above it? This is something non-obvious and usually that's a good reason for a comment.